### PR TITLE
fixes the Jenkins chart old version dependancies

### PR DESCRIPTION
**Pull request by ahmedmansour5**
This updates the Jenkins chart version / dependencies used in the mushop-setup namespace

### CODE REVIEW

File: deploy/complete/helm-chart/setup/values.yaml [hunk 1]
```diff
@@ -123,31 +123,35 @@ grafana:
...
       enabled: true
-      paths:
-        - path: /jenkins
-          backend:
-            serviceName: >-
-              {{ template "jenkins.fullname" . }}
-            servicePort: 8080
-      apiVersion: "networking.k8s.io/v1beta1"
+      apiVersion: networking.k8s.io/v1
+      #apiVersion: "networking.k8s.io/v1beta1"
```
Comment (junior) [inline deploy/complete/helm-chart/setup/values.yaml:137]: Remove the comments

File: deploy/complete/helm-chart/setup/values.yaml [hunk 2]
```diff
@@ -123,31 +123,35 @@ grafana:
...
       labels: {}
-      annotations:
-        kubernetes.io/ingress.class: nginx
```
Comment (junior) [inline deploy/complete/helm-chart/setup/values.yaml:144]: The annotations: 
```
    kubernetes.io/ingress.class: nginx
    nginx.ingress.kubernetes.io/rewrite-target: /$2
```
are necessary to get the right ingress controller. The second annotations may be needed to make sure that everything after /jenkins works. You need to test.

File: deploy/complete/helm-chart/setup/values.yaml [hunk 3]
```diff
@@ -123,31 +123,35 @@ grafana:
...
+      spec:
+        ingressClassName: nginx
```
Comment (junior) [inline deploy/complete/helm-chart/setup/values.yaml:140]: If use the ingress.class annotation, the ingressClassName need to be removed

File: deploy/complete/helm-chart/setup/values.yaml [hunk 4]
```diff
@@ -123,31 +123,35 @@ grafana:
...
-      - configuration-as-code:1.41
-      - blueocean:1.23.2
-    
+        - kubernetes:3900.va_dce992317b_4
+        - workflow-aggregator:596.v8c21c963d92d
+        - git:5.0.1
+        - configuration-as-code:1625.v27444588cc3d
+        - blueocean:1.27.3
+        - credentials-binding:604.vb_64480b_c56ca_
+        - workflow-job:1292.v27d8cc3e2602
```
Comment (junior) [inline deploy/complete/helm-chart/setup/values.yaml:157]: the last line need to be empty

**Review by junior — CHANGES_REQUESTED** [review]

**Comment by junior** [comment]
You can use the Grafana ingress as reference: [deploy/complete/helm-chart/mushop/templates/ingress-grafana.yaml](https://github.com/oracle-quickstart/oci-cloudnative/blob/master/deploy/complete/helm-chart/mushop/templates/ingress-grafana.yaml)

**Comment by ahmedmansour5** [comment]
Check the new updates and let me know if it has the requested changes.

**Review by junior — APPROVED** [review]
LG

**Pull request merged**